dev/breeze/doc/images/output_testing_db-tests.svg

```diff
@@ -0,0 +1,117 @@
+<svg class="rich-terminal" viewBox="0 0 1482 391.59999999999997" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Drop this file. It must have been added before rebase / latest changes with redesigning the test commands.

Reply:
Ah good eye. Weird.

Reply:
it's the pre-commit that spotted it - not me :).

BTW. And now you need to re-run pre-commit (`breeze static-check --only-my-changes` as well as it needs to regenerate the images with PostgreSQL 12 removed :(

Reply:
My breeze was in a pretty unhappy state somehow - it had removed 12 from a few but not all. But should be good now. Need to move `breeze down` higher on my troubleshooting list...